packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -59,6 +59,14 @@ export function useFetch<
   if (!request) {
     throw new Error('[nuxt] [useFetch] request is missing.')
   }
+
+  if (typeof request === 'function') {
+    const requestUrl = request() as string
+    if (requestUrl.startsWith('//')) {
+      throw new Error('[nuxt] [useFetch] the request URL must not start with "//".')
+    }
+  }
```

Comment:
maybe move this to L123 and test `_request.value` instead?

Reply:
@danielroe I moved the logic to line 123. The reason I had it at line 63 earlier was because I wanted to keep all the "throw errors" together.

Now it works okay, but I cannot figure out why the error does not appear nicely on the screen when thrown. It kind of throws an error silently. This is not the case if I move the logic back to line 63.

Reply:
Good point - it's being captured and included in `error.value` but that's a little too hidden. We could move it up then, just after the creation of `_request` so we don't have to normalise the request twice.